<?php

namespace App\Gpp\Responsibles\Exceptions;

use Exception;

class ResponsibleNotFoundException extends Exception
{
    /**
     * Report or log an exception.
     *
     * @param  \Throwable  $exception
     * @return void
     *
     * @throws \Throwable
     */
    public function report()
    {
        // 
    }

    /**
     * Render an exception into an HTTP response.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Throwable  $exception
     * @return \Symfony\Component\HttpFoundation\Response
     *
     * @throws \Throwable
     */
    public function render($request)
    {
        $response = ['message'=> "Ce responsable est introuvable"];
        return response()->json($response,422);
    }
}
